src/pycsspeechtts/pycsspeechtts.py

```diff
@@ -73,7 +73,7 @@ def name_lang(language):
             prosody.set('volume', volume)
             prosody.set('pitch', pitch)
             prosody.set('contour', contour)
-            prosody.text = text
+            prosody.append(ElementTree.fromstring(text))
```

Comment:
```suggestion
            voice.append(ElementTree.XML('<prosody>'+text+'</prosody>'))
            prosody = voice.find('prosody')
            prosody.set('rate', rate)
            prosody.set('volume', volume)
            prosody.set('pitch', pitch)
            prosody.set('contour', contour)
```

I had the same request: being able to use embedded XML. The lines of code above allows for using XML tags inside the text, whilst also remaining backward compatible with a regular string. 

For example, all of these work:

- `This is a regular, backwards compatible sentence`
- `This is a sentence with a <break time="2s" /> long pause`
- `<p>Sentence 1 in paragraph 1.</p><p>Sentence 2 in paragraph 2</p>`

(oh and to be clear: this replaces lines 71 up to and including 76)